src/stories/SegmentedPicker/index.tsx

```diff
@@ -0,0 +1,63 @@
+import { FC, useState } from "react";
+import clsx from "clsx";
+import Text from "../Typography/Text";
+import { SegmentedPickerProps, Segment } from "./type";
+
+export const SegmentedPicker: FC<SegmentedPickerProps<any>> = ({
+  className,
+  segments,
+  onSelect,
+}) => {
+  const [selectedSegment, setSelectedSegment] = useState<Segment<any> | null>(
```

Comment:
please provide a prop named value，and if provide value，change this useState default value

Reply:
change:
  const [selectedSegment, setSelectedSegment] = useState<Segment<T> | null>(
    segments.find((segment) => segment.value === value) || null
  );